lib/extend/tag.js

```diff
@@ -5,7 +5,8 @@ const { cyan } = require('chalk');
 const { Environment } = require('nunjucks');
 const Promise = require('bluebird');
 const placeholder = '\uFFFC';
-const rPlaceholder = /(?:<|&lt;)!--\uFFFC(\d+)--(?:>|&gt;)/g;
+const rPlaceholder = /<!--\uFFFC(\d+)-->/g;
```

Comment:
I wonder why the previous regex matches `&lt;` `&gt;`? I thought `<>` is only escaped in [code block](https://github.com/hexojs/hexo-util/blob/cd6dcfda34f39cbf992c1e20bb1d4d5570ec7b3c/lib/highlight.js#L104)?

Reply:
@curbengh IMHO, the regexp here might be copied from `post.js` in which the `<>` will be escaped by post renderer (like marked.js).

But inside tag rendering process, there will be no escape at all.